### PR TITLE
reserved-ipv6: Add reserved IPv6 commands

### PR DESCRIPTION
**Pull request by amanoj-do**
This PR allows doctl users to interact with Reserved IPv6 akin to existing Reserved IPs

Note: Reserved IPv6 is a private beta feature and not available for public use.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.22
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/creack/pty v1.1.21
-	github.com/digitalocean/godo v1.131.0
+	github.com/digitalocean/godo v1.131.1-0.20241127211050-a97e39731918
```
Comment (loosla) [inline go.mod:8]: :wave: Could we use [v1.131.0](https://github.com/digitalocean/godo/releases/tag/v1.131.0) as required changes already should be there :pray: 

> https://github.com/digitalocean/godo/pull/759 - @imaskm - Add reserved ipv6 changes as Beta

Comment (loosla) [reply go.mod:8]: Nice :raised_hands:  Thanks for explanation :+1:

File: commands/reserved_ipv6s.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+	cmdReservedIPv6Create := CmdBuilder(cmd, RunReservedIPv6Create, "create", "Create a new reserved IPv6 address", `Creates a new reserved IPv6 address.
+Reserved IPv6 addresses can be held in the region they were created in on your account.`, Writer,
+		aliasOpt("c"), displayerType(&displayers.ReservedIPv6{}))
+	AddStringFlag(cmdReservedIPv6Create, doctl.ArgRegionSlug, "", "", "The region where to create the reserved IPv6 address.")
+	cmdReservedIPv6Create.Example = `The following example creates a reserved IPv6 address in the ` + "`" + `nyc1` + "`" + ` region: doctl compute reserved-ipv6 create --region nyc1`
+
+	cmdReservedIPv6Get := CmdBuilder(cmd, RunReservedIPv6Get, "get <reserved-ipv6>", "Retrieve information about a reserved IPv6 address", "Retrieves detailed information about a reserved IPv6 address, including its region and the ID of the Droplet its assigned to.", Writer,
+		aliasOpt("g"), displayerType(&displayers.ReservedIPv6{}))
+	cmdReservedIPv6Get.Example = `The following example retrieves information about the reserved IPv6 address ` + "`" + `5a11:a:b0a7` + "`" + `: doctl compute reserved-ip get 5a11:a:b0a7`
```
Comment (loosla) [inline commands/reserved_ipv6s.go:48]:
```suggestion
	cmdReservedIPv6Get.Example = `The following example retrieves information about the reserved IPv6 address ` + "`" + `5a11:a:b0a7` + "`" + `: doctl compute reserved-ipv6 get 5a11:a:b0a7`
```

File: commands/reserved_ipv6s.go [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+	cmdRunReservedIPv6Delete := CmdBuilder(cmd, RunReservedIPv6Delete, "delete <reserved-ipv6>", "Permanently delete a reserved IPv6 address", "Permanently deletes a reserved IPv6 address. This is irreversible.", Writer, aliasOpt("d", "rm"))
+	AddBoolFlag(cmdRunReservedIPv6Delete, doctl.ArgForce, doctl.ArgShortForce, false, "Deletes the reserved IPv6 address without confirmation")
+	cmdRunReservedIPv6Delete.Example = `The following example deletes the reserved IPv6 address ` + "`" + `5a11:a:b0a7` + "`" + `: doctl compute reserved-ip delete 5a11:a:b0a7`
```
Comment (loosla) [inline commands/reserved_ipv6s.go:52]:
```suggestion
	cmdRunReservedIPv6Delete.Example = `The following example deletes the reserved IPv6 address ` + "`" + `5a11:a:b0a7` + "`" + `: doctl compute reserved-ipv6 delete 5a11:a:b0a7`
```

File: commands/reserved_ipv6s.go (hunk 2 above)
Comment (amanoj-do) [reply commands/reserved_ipv6s.go:48]: nice catch! Thanks 🙇

**Comment by amanoj-do** [comment]
👋 @loosla , Let me know if you need anything else from our side 🙇

**Review by loosla — APPROVED** [review]
Looks great! Thanks :+1:

**Pull request merged**